scripts/feature_tests/setup_env.sh

```diff
@@ -9,5 +9,10 @@ if [[ "${1}" == "ug" ]]; then
   # shellcheck source="$M3PATH/scripts/feature_tests/upgrade/upgrade_vars.sh"
   source "${M3PATH}/scripts/feature_tests/upgrade/upgrade_vars.sh"
 fi
+if [[ "${1}" == "no-tls" ]]; then
```

Comment:
we do not need this file.
we are ok with TLS enabled for now.

Reply:
to be cleaned up later.

Reply:
It will not make any change to the current features.